config/environments/development.rb

```diff
@@ -12,9 +12,11 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
-  # Show full error reports and disable caching.
+  # Set this to +true+ unless you want to skip exceptions_app:
+  config.consider_all_requests_local = false
```

Comment:
Can we make this configurable, `true` by default? This is what enables Rails' default error pages in development, which provide more detail than our own error logging.

Reply:
@ArtOfCode- tbf, our error reports are quite good - I've never had to actually wade into the extended stack to resolve one. That said, I have no issue with making this configurable - I am not sure that the default should be `true` as it basically disables `ErrorsController` without the user realizing it, but have no explicit preference.

Reply:
Done, @ArtOfCode- - I alresdy decided not to change the default when making `consider_all_requests_local` configurable, so it was only a matter of adding a note to the controller